app/controllers/basechurch/v1/bulletins_controller.rb

```diff
@@ -1,11 +1,27 @@
 class Basechurch::V1::BulletinsController < Basechurch::ApplicationController
   before_action :authenticate_user!, except: [:show, :sunday]
+  before_action :set_signing_expiry, only: :sign
 
   def sunday
     params[:id] = fetch_sunday_bulletin_id.to_s
     show
   end
 
+  def sign
+    render json: {
+      acl: 'public-read',
+      awsAccessKeyId: Rails.application.secrets.aws_access_key_id,
+      bucket: 'mcac-staging',
+      'Cache-Control' => 'max-age=630720000, public',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.